README.md

```diff
@@ -92,7 +104,8 @@ Installing certificates, and configuring the docker overlay:
 4. Make certs:
     1. `cd ~/.dinghy/certs`
     2. `mkcert -cert-file app.lara.docker.crt -key-file app.lara.docker.key app.lara.docker`
-    3. `mkcert -cert-file app.portal.docker.crt -key-file app.portal.docker.key app.portal.docker`
+       [For automation `mkcert -cert-file authoring.dev.docker.crt -key-file authoring.dev.docker.key authoring.dev.docker`]
+    3. `mkcert -cert-file learn.dev.docker.crt -key-file learn.dev.docker.key learn.dev.docker` 
```

Comment:
```suggestion
    3. `mkcert -cert-file app.portal.docker.crt -key-file app.portal.docker.key app.portal.docker` 
    4. For automation:
        1.  `mkcert -cert-file authoring.dev.docker.crt -key-file authoring.dev.docker.key authoring.dev.docker`
        2.  `mkcert -cert-file learn.dev.docker.crt -key-file learn.dev.docker.key learn.dev.docker`
```

For a developer following the regular readme setup for lara and the portal the'll still want a cert for `app.portal.docker`